src/components/Grid/index.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { GridItem, GridLayout } from './Grid.styled';
+
+const Grid = ({ children, col }) => {
+    return(
+        <GridLayout maxColumnCount={col}>
+            {React.Children.toArray(children).map((item) => {
+                return (
```

Comment:
You can use shorthand here, instead () => { return (<GridItem />) }, just use () => (<GridItem />)

Reply:
Fixed